clowdapp.yaml

```diff
@@ -306,3 +308,5 @@ parameters:
   value: 'INFO'
 - name: PARTITION_DELETE_INTERVAL
   value: "0 0 */15 * *" # Runs at 12:00 AM, every 15 days.
+- name: ID_PROVIDER
```

Comment:
was this part pf the change tested? I guess not relevant in our context, but when merged back, perhaps worth adding a comment about it.